src/Chainweb/Pact/Backend/Types.hs

```diff
@@ -141,6 +155,33 @@ instance FromJSON PactDbConfig
 
 makeLenses ''PactDbConfig
 
+data SQLiteRowDelta = SQLiteRowDelta
```

Comment:
Haddock all of these: SQLite[RowDelta,PendingWrites,PendingData] in order to  describe their semantics

Reply:
done